corehq/apps/app_manager/templatetags/xforms_extras.py

```diff
@@ -181,7 +187,7 @@ def inline_edit_trans(name, langs=None, url='', saveValueName='', postSave='',
     return format_html(template, **options)
 
 
-def _get_dynamic_input_trans_options(name, langs=None, allow_blank=True, is_textarea=False):
+def _get_dynamic_input_trans_options(name, escape_method: EscapeMethod, langs=None, allow_blank=True):
```

Comment:
Type hint? Not sure if we want to open that can of worms :). (for the record I would be for them)

Reply:
yaaa personally i find it useful for stuff like this, otherwise there's no clear way to know what should be passed to `escape_method`